### PR TITLE
Slashes through entities, simplified

### LINKED ISSUE #25: Can't use slash or comma in names
Slashes and commas are reserved as path and column delimiters, respectively. This prevents their use in names (path parts). Or rather, it leads to unexpected results.

My particular use case is in [dirstat](https://github.com/mlange-42/dirstat) again, where I would like to use a trailing slash to denote directories.

I see two options to solve this:

* Configurable column and path delimiters
* Using HTLM entities, like `&sol;` for these characters
  * Comes with some problems:
    * Would not work well with string length calculations for box titles
      * Could use regex or a dedicated XML/HTML encoding/decoding library to fix this
    * `&` is currently replaced by `&amp;` to form valid XML/SVG
      * Would require to take care of HTML entities in `&` replacement

### PR DESCRIPTION
**Pull request by mlange-42**
Simply replace entity for slash by slash when contructing UIText.

Fixes #25

### CODE REVIEW

**Review by nikolaydubina — CHANGES_REQUESTED** [review]
looks better!

could you add test file input and SVG output? we need to freeze this behavior into a test case

**Comment by mlange-42** [comment]
> looks better!
> 
> could you add test file input and SVG output? we need to freeze this behavior into a test case

Looks like the SVGs are created by the `Makefile`. But how is the actual comparison/test done?

**Comment by mlange-42** [comment]
Done as far as I understood how it works.

File: render/render.go [hunk 1]
```diff
@@ -16,6 +17,10 @@ const (
 	textMarginH          float64 = 2
 )
 
+var entityToSlash = strings.NewReplacer(
```
Comment (nikolaydubina) [inline render/render.go:20]:
```suggestion
// entityToSlash has HTML entities to strings mapping
var entityToSlash = strings.NewReplacer(
```

**Comment by nikolaydubina** [comment]
yes, this is pretty much it. just manually verify that SVG makes sense

**Review by nikolaydubina — APPROVED** [review]

**Pull request merged**